noir-projects/noir-protocol-circuits/crates/types/src/meta/mod.nr

```diff
@@ -104,8 +104,8 @@ pub comptime fn generate_deserialize_from_fields<N>(
             }
             let packed_fields = packed_fields_quotes.join(quote {,});
 
-            // No we call unpack on the type
-            result = quote { aztec::protocol_types::traits::Packable::unpack([ $packed_fields ]) };
```

Comment:
This is the reason why the Packable needs to be imported now.

Having the full path here was a mistake since the dep could be called differently than `aztec` and this might also be used inside the protocol circuits where there is no aztec dep.

We probably could figure the correct import at comptime but would do that in a separate PR if we find this desirable.

Reply:
Ahh, I see. Can't we have the full path to proto_types::traits so that it'd work both here, in aztec, and in the user's code?

Reply:
You would then need to figure out whether you are in the types crate or not as in the types crate you would not use `dep::aztec::protocol_types` but just `crate`.

Jake promised me he will send a PR such that I can quote a Type directly and get a correct path. This would then make this straighforward. Until that is done I've created [this issue](https://github.com/AztecProtocol/aztec-packages/issues/11602)to track it.